### PR TITLE
Add MAlert components 

### PR DESCRIPTION
**Pull request by nigiiro-taro**
We can set type, title at MAlert's body
when we want to set List, We can set description, list, listsize.

### Story of MAlert
スクリーンショット 2021-06-22 10 35 11

### CODE REVIEW

File: src/components/molecules/Alert/types.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+export interface Props {
+  /**
+   * TODO: 仮のプロパティ名を設定しています。
+   *       API連携時に正しいプロパティ名に変更します。
+   */
```
Comment (shskwmt) [inline src/components/molecules/Alert/types.ts:5]: TODOはそのまま残しますか？

File: src/components/molecules/Alert/Alert.stories.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    return {
+      alertList: ['企業情報', '企業の魅力']
+    };
+  }
+});
+
+export const Default = Template.bind({});
+Default.args = {
+  title: 'クライアントから承認依頼がきています',
+  type: 'draft'
```
Comment (shskwmt) [inline src/components/molecules/Alert/Alert.stories.js:34]: もう少し汎用的な例にした方がよいと思います

**Review by shskwmt — CHANGES_REQUESTED** [review]
コメントしました

File: src/components/molecules/Alert/Alert.vue [hunk 3]
```diff
@@ -0,0 +1,128 @@
+<template>
+  <div class="alert" :class="`${alertType}`">
+    <div class="alert__title">
+      <div class="alert__icon">
+        <MIcon icon-name="lightbulb" class="alert__icon-fontawesome" />
+      </div>
+      <div v-if="title.length">
```
Comment (mharaknight) [inline src/components/molecules/Alert/Alert.vue:7]:
```suggestion
      <div v-if="title">
```
こうですかね

Comment (mharaknight) [inline src/components/molecules/Alert/Alert.vue:7]:
```suggestion
      <div v-if="title">
```
length is not necessary so title is string and not array

File: src/components/molecules/Alert/Alert.vue [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+    list: {
+      type: Array,
+      default: () => []
+    },
+    listsize: {
+      type: String,
+      default: 'small'
+    }
+  },
+  data() {
```
Comment (mharaknight) [inline src/components/molecules/Alert/Alert.vue:64]: data is empty
please delete it

**Review by mharaknight — COMMENTED** [review]
commented

File: src/components/molecules/Alert/types.ts (hunk 1 above)
Comment (nigiiro-taro) [reply src/components/molecules/Alert/types.ts]: deleted

File: src/components/molecules/Alert/Alert.stories.js (hunk 2 above)
Comment (nigiiro-taro) [reply src/components/molecules/Alert/Alert.stories.js]: I changed example. Please check them.

b4cce72

File: src/components/molecules/Alert/Alert.vue (hunk 3 above)
Comment (nigiiro-taro) [reply src/components/molecules/Alert/Alert.vue]: deleted

File: src/components/molecules/Alert/Alert.vue (hunk 4 above)
Comment (nigiiro-taro) [reply src/components/molecules/Alert/Alert.vue]: deleted

b4cce72

**Comment by nigiiro-taro** [comment]
I Change the name from alert to MAlert. 
I've already confirmed that it works with mina.

**Review by mharaknight — APPROVED** [review]
LGTM

**Review by shskwmt — APPROVED** [review]

**Pull request merged**